### PR TITLE
Feat/unsupported network modal

### PR DESCRIPTION
**Pull request by alvaro-ricotta**
Adding modal message for Unsupported network :
When a user changes the network directly from MetaMask, we display a modal indicating that the chosen network is not supported, and we give them the option to automatically connect to some of the networks we support.

### CODE REVIEW

File: packages/boba/gateway/src/hooks/useNetwork.ts [hunk 1]
```diff
@@ -20,6 +20,7 @@ import { selectActiveNetwork, selectActiveNetworkType } from 'selectors'
 import { NetworkList } from 'util/network/network.util'
 
 const useNetwork = () => {
+  console.log('here')
```
Comment (sk-enya) [inline packages/boba/gateway/src/hooks/useNetwork.ts]: please remove console.

**Comment by sk-enya** [comment]
@alvaro-ricotta not sure but seem like this change is already handled in 
https://github.com/bobanetwork/boba/pull/1206 can you please confirm once and close this?

**Comment by alvaro-ricotta** [comment]
Hey @sk-enya this change include the unsupported modal, this have to be merged first, before than the other https://github.com/bobanetwork/boba/pull/1206

**Review by sk-enya — APPROVED** [review]
Looks good!

**Pull request merged**